src/local-run.ts

```diff
@@ -121,15 +119,18 @@ export const runWithOutgoingDomains = async function (
 
   const command = getCommandline(
     Deno.mainModule,
-    denoExecutablePath,
     manifest,
     devDomain,
     hookCLI,
   );
 
-  const p = Deno.run({ cmd: command });
+  const commander = new Deno.Command(denoExecutablePath, {
+    args: command,
+  });
+
+  const subprocess = commander.spawn();
+  const status = await subprocess.status;
```

Comment:
One thing we should test around this is whether or not this leaves the process open or if we should close via something like `subprocess.kill()`

Reply:
Generally, once the status code from a process is able to be retrieved, this implies that the process completed, so my expectation is that we don't need to do that - but doesn't hurt!